api/store/mongo/migrations/migration_38.go

```diff
@@ -0,0 +1,53 @@
+package migrations
+
+import (
+	"context"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var migration38 = migrate.Migration{
+	Version:     38,
+	Description: "Set last_login to created_at, when created_at is a zero value",
+	Up: func(db *mongo.Database) error {
+		logrus.WithFields(logrus.Fields{
+			"component": "migration",
+			"version":   38,
+			"action":    "Up",
+		}).Info("Applying migration")
+		zeroTime := time.Time{}.UTC()
+		_, err := db.Collection("users").Aggregate(context.TODO(),
+			mongo.Pipeline{
+				{
+					{"$match", bson.D{
+						{"created_at", zeroTime},
+					}},
+				},
+				{
+					{"$replaceRoot", bson.D{{"newRoot", bson.M{"$mergeObjects": bson.A{"$$ROOT", bson.M{"created_at": "$last_login"}}}}}},
+				},
+				{
+					{"$out", "users"},
```

Comment:
This line terminate the mongo's pipeline saying to write modified document. However, there is another staged, `$merge`, that could be used in the same way, in this case.

https://docs.mongodb.com/manual/reference/operator/aggregation/out/#mongodb-pipeline-pipe.-out